src/Footer.module.css

```diff
@@ -0,0 +1,237 @@
+.footer {
+  margin-top: 0.5rem;
+  padding: 1rem 0;
+  text-align: center;
+  color: #f2e5de;
+  background-color: #1a1a1aa9;
+  border-radius: 0.5rem;
+
+  a {
```

Comment:
Note to self, warning in build

```
warnings when minifying css:
Warning: G] A nested style rule cannot start with "a" because it looks like the start of a declaration [css-syntax-error]

    <stdin>:[15](https://github.com/Cellule/dndGenerator/actions/runs/14423219728/job/40448403345?pr=209#step:5:16)9:2:
      159 │   a {
          │   ^
          ╵   :is(a)

  To start a nested style rule with an identifier, you need to wrap the identifier in ":is(...)" to prevent the rule from being parsed as a declaration.
```